tests/View/Mix/MixTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Illuminate\Tests\View\Mix;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Support\HtmlString;
+
+class MixTest extends TestCase
+{
+    public function setUp()
+    {
+        app()->singleton('path.public', function () {
+            return __DIR__;
+        });
+    }
+
+    private function getMix()
+    {
+        return new \Illuminate\View\Mix\Mix;
```

Comment:
Ok Lucas.
So, I'll do all the modifications tonight. Do you see now something else to update? I think it will be the last time, this is THE moment to tell me! 🙃